### DISCUSSION TITLE
How to search-replace a "constant" string with renovate

### DISCUSSION THREAD

**Discussion by ppussar** [renovatebot/renovate#18174] [Request Help]
Hi there,

is it possible to search and replace a certain string with renovate? My usecase:
I need to adapt a config value in one file for all of my projects. I am wondering if renovate could do the job, e.g. with the regexManager?

But I am not sure if I need to configure some special datasource in that case?

Thanks and best regards,
Patrick

**Comment by viceice** [comment]
you always need a datasource to update a value with renovate

**Reply by ppussar** [reply]
is it possible to define some kind of constant datasource, which always provides predefined (in config) values? Would it makes sense to implement this if it is not available yet?

Or are there some creative solutions for this use-case, e.g. by abusing some available datasource to trigger the regex manager?
